code/modules/mining/machine_silo.dm

```diff
@@ -92,39 +124,35 @@ GLOBAL_LIST_EMPTY(silo_access_logs)
 	return materials.ui_static_data()
 
 /obj/machinery/ore_silo/ui_data(mob/user)
-	var/list/data = list(
-		"materials" =  materials.ui_data()
-	)
+	. = list()
```

Comment:
Revert back to using ``var/list/data`` please, that's what our standards say to use